guides/2022-04-05-guide-to-conduit-connector-lifecycle.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: "The Guide to the Conduit Connector Lifecyle"
+slug: guide-to-conduit-connector-lifecycle
+author: '@anaptfox'
+author_url: https://twitter.com/anaptfox
+author_image_url: https://pbs.twimg.com/profile_images/1004707739405205504/BpvzOr9X_400x400.jpg
+tag: ['File Connector']
+description: Learn how to build a Conduit Algolia Connector
+keywords: ['data', 'connector', 'algolia', 'conduit']
+---
+
+# The Guide to the Conduit Connector Lifecyle
```

Comment:
I'll be brutally honest here, I don't see a lot of added value in this article compared to what is already described in the SDK docs for the [`Source`](https://pkg.go.dev/github.com/conduitio/conduit-connector-sdk#Source) and [`Destination`](https://pkg.go.dev/github.com/conduitio/conduit-connector-sdk#Destination) interfaces. As a developer, I would probably rather look at the go doc reference instead of this article, given that they contain the same information.

Reply:
My question here would be: How would a developer know how to find that information?

Reply:
I generally look for the Go doc badge that's normally on top of the readme. Or I `go get` the library and explore it using the IDE.

Reply:
I kind of agree with @lovromazgon of the guides this is the weakest and feels a little light. Instead of what's essentially a recap of the words in the docs, maybe we go the route of illustrating the lifecycles of source vs destination and break down the article like that. Two illustrations and then a walk through of what's happening in the lifecycle of a source or destination.